README.md

```diff
@@ -111,6 +116,7 @@ LLNL-CODE-766763
 - Siu Wun Cheung (LLNL)
 - Eric Chin (LLNL)
 - Youngsoo Choi (LLNL)
+- Seung Whan Chung (LLNL)
```

Comment:
Can we use the below?
"Kevin" Seung Whan Chung

Reply:
Sure thing! :)